### PR TITLE
Ci image changes

### PR DESCRIPTION
**Pull request by joelrebel**
### What does this PR do

Repo maintenance

 - Updates the linter to the latest.
 - Updates Go version to 1.19 and all deps to current latest.
 - Removes ironlib docker image builds and CI build jobs.

### CODE REVIEW

File: utils/dsu.go [hunk 1]
```diff
@@ -284,7 +284,8 @@ func findDSUInventoryCollector(path string) []string {
 // since the component name exposed by the dsu command doesn't tell the component name in a unique manner,
 // the model.DellComponentSlug list has be ordered to ensure we don't have incorrect identification.
 // Attempts were made to use fuzzy matching and levenstiens distance,
-//  to identify the components correctly, although none seemed to work as well as an ordered list.
+//
```
Comment (splaspood) [inline utils/dsu.go:287]: I think this extra blank commented line is a typo.

File: utils/smc_ipmicfg.go [hunk 2]
```diff
@@ -26,7 +26,8 @@ type IpmicfgSummary struct {
 
 // Return a new Supermicro IPMICFG executor
 // note: the binary is expected to be available as smc-ipmicfg,
-//       as setup in the fup firmware-update image
+//
```
Comment (splaspood) [inline utils/smc_ipmicfg.go:29]: I think this extra blank commented line is a typo.

**Review by splaspood — APPROVED** [review]
Looks good...  A couple comment formatting issues which aren't a big enough deal to block merge IMHO.

File: utils/dsu.go (hunk 1 above)
Comment (joelrebel) [reply utils/dsu.go]: Thanks, fixed now. Running gofmt added those for some reason.

**Review by splaspood — APPROVED** [review]

**Pull request merged**